Makefile

```diff
@@ -92,10 +92,10 @@ test_go_race:
 	$(go_test) ./... -race
 
 test_sharness_short:
-	cd test/sharness/ && make
+	make -C test/sharness/
 
 test_sharness_expensive:
-	cd test/sharness/ && TEST_EXPENSIVE=1 make
+	TEST_EXPENSIVE=1 make -C test/sharness/
```

Comment:
Yeah, it's fine. One could also use `make -C test/sharness/ TEST_EXPENSIVE=1`. It's a matter of preference whether one wants the shell or `make` to do the `cd` and to set the environment variable.